strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/configuration/StoragesConfigurationController.java

```diff
@@ -305,7 +305,7 @@ public ResponseEntity getRepositoryResponseEntity(@RepositoryMapping Repository
     @PreAuthorize("hasAuthority('CONFIGURATION_DELETE_REPOSITORY')")
     @DeleteMapping(value = "/{storageId}/{repositoryId}", produces = { MediaType.TEXT_PLAIN_VALUE,
                                                                        MediaType.APPLICATION_JSON_VALUE })
-    public ResponseEntity removeRepository(@RepositoryMapping Repository repository,
+    public ResponseEntity removeRepository(@RepositoryMapping(allowOutOfServiceRepository = true) Repository repository,
```

Comment:
This change needs to be applied to the `getRepositoryResponseEntity` method as well.

Reply:
Done.